src/lib/override_cursor.js

```diff
@@ -0,0 +1,43 @@
+/**
+* Copyright 2012-2017, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+
+'use strict';
+
+var setCursor = require('./setcursor');
+
+var STASHATTR = 'data-savedcursor';
+
+/*
+ * works with our CSS cursor classes (see css/_cursor.scss)
+ * to override a previous cursor set on d3 single-element selections,
+ * by moving the name of the original cursor to the data-savedcursor attr.
+ * omit cursor to revert to the previously set value.
+ */
+module.exports = function overrideCursor(el3, csr) {
```

Comment:
It would be nice to a few test cases for `overrideCursor` similar to the ones for `setCursor` [here](https://github.com/plotly/plotly.js/blob/e2105d2d587168baf24fe8bd4cb113f29ec97c50/test/jasmine/tests/lib_test.js#L1145).